src/pages/CharacterSheetPage.tsx

```diff
@@ -0,0 +1,255 @@
+import React, { useState, useEffect, ChangeEvent } from "react";
+import {
+  Box,
+  Input,
+  Textarea,
+  Checkbox,
+  VStack,
+  Grid,
+  GridItem,
+  FormControl,
+  FormLabel,
+  Heading,
+  HStack,
+  Divider,
+  Button,
+} from "@chakra-ui/react";
+import { useSearchParams, useNavigate } from "react-router-dom";
+import { v4 as uuidv4 } from "uuid"; // 🔹 Per generare un ID univoco se necessario
+
+interface CharacterSkills {
+  [key: string]: { proficiency: boolean; expertise: boolean };
+}
+
+interface CharacterState {
+  id: string;
+  name: string;
+  class: string;
+  level: number;
+  background: string;
+  race: string;
+  age: number;
+  features: string;
```

Comment:
```suggestion
  features: string[];
```
Non dovrebbe essere un array?